src/graph.js

```diff
@@ -39,11 +55,12 @@ class Graph extends React.Component {
     render() {
         const branches = this.state.branches
         return (
-            <div>
+            <React.Fragment>
                 <form
                     onSubmit={(e) => {
                         e.preventDefault();
                         this.addBranch();
+
                     }}>
```

Comment:
Can the forms-section be a component on its own? That would be helpful to reduce the complexity of the `render()` function.

Reply:
I will try to make it a separate component

Reply:
Resolving this since @aryanbaburajan will take over this